tools/openms/CompNovo.xml

```diff
@@ -41,6 +41,11 @@
 #end if
 #if $param_algorithm_estimate_precursor_mz:
   -algorithm:estimate_precursor_mz
+  #if " " in str($param_algorithm_estimate_precursor_mz):
```

Comment:
The only difference between if/else are the "quotes"?
The if condition can also be never true anymore? Is this all due to the converter?
Without looking much further I think you can remove the entire if and write `-algorithm:estimate_precursor_mz $param_algorithm_estimate_precursor_mz`